go.mod

```diff
@@ -1,77 +1,76 @@
 module github.com/crossplane-contrib/function-dummy
 
-go 1.21
+go 1.24
 
-toolchain go1.21.3
+replace github.com/crossplane/function-sdk-go => github.com/negz/function-sdk-go v0.0.0-20250711051713-67cc65c5665e
```

Comment:
Depends on https://github.com/crossplane/function-sdk-go/pull/212.

Need to release that and remove this before merging.

Reply:
Done, thanks!